apps/passport-server/src/services/generic-issuance/pipelines/PretixPipeline.ts

```diff
@@ -7,37 +52,723 @@ import {
   PipelineType
 } from "./types";
 
+const LOG_NAME = "PretixPipeline";
+const LOG_TAG = `[${LOG_NAME}]`;
+
+/**
+ * This object represents a configuration from which the server can instantiate
+ * a functioning {@link PretixPipeline}. It's entirely specified by the user.
```

Comment:
one note is that in retrospect `It's entirely specified by the user.` is not entirely accurate. in particular, one thing we should be careful about is not allowing the user to specify `genericIssuanceId` for the event/product ids, because that would enable them to forge tickets to other events. another way it's inaccurate is depending on how we implement the loading functionality, if we save product/event names in the pipeline options, those will need to be updated by our code over time.